signSDK/sign/src/main/kotlin/com/walletconnect/sign/client/mapper/ClientMapper.kt

```diff
@@ -206,4 +210,14 @@ internal fun EngineDO.InternalError.toClientError(): Sign.Model.Error =
 @JvmSynthetic
 internal fun String.strippedUrl() = Uri.parse(this).run {
     this@run.scheme + "://" + this@run.authority
+}
+
+@JvmSynthetic
+internal fun String.addUserAgent(): String {
+    return Uri.parse(this).buildUpon()
+        .appendQueryParameter("protocol","wc-relay-2.0")
+        .appendQueryParameter("sdk", "kotlin-2.0-rc.0") // Setup env variable for version and tag. Use env variable here instead of hard coded version
```

Comment:
```suggestion
        .appendQueryParameter("sdk", "kotlin-2.0-rc.0") // TODO: Setup env variable for version and tag. Use env variable here instead of hard coded version
```
Just for better search via todo tag